Git Formats/Git Commit.sublime-syntax

```diff
@@ -112,12 +112,18 @@ contexts:
         1: keyword.other.change-list.git.commit
         2: punctuation.separator.mapping.pair.change-list.git.commit
         3: string.unquoted.git.commit markup.inserted.file.git.commit
-    - match: \b({{copied}}|{{renamed}}|{{typechange}})\s*(:)\s*(.*)
+    - match: \b({{copied}}|{{renamed}}|{{typechange}})\s*(:)\s*
       scope: meta.change-list.git.commit
       captures:
         1: keyword.other.change-list.git.commit
         2: punctuation.separator.mapping.pair.change-list.git.commit
-        3: string.unquoted.git.commit markup.changed.file.name.git.commit
+      push:
+        - meta_scope: meta.change-list.git.commit string.unquoted.git.commit markup.changed.file.name.git.commit
```

Comment:
The `meta_scope` introduces a regression, which causes the keywords `copied`, `renamed` or `typechange` to be scoped as `string.unquoted.git.commit markup.changed.file.name.git.commit`.

You'd better use:

```YAML
- meta_scope: meta.change-list.git.commit
- meta_content_scope: string.unquoted.git.commit markup.changed.file.name.git.commit
```

As comments are unlikely to be modified by the user and the expression will only contain one single `->`, pushing into a context might not be required at all. The following would work well enough.

```YAML
    - match: \b({{copied}}|{{renamed}}|{{typechange}})\s*(:)\s*((.+)\s+(->)\s+(.+)|(.+))
      scope: meta.change-list.git.commit
      captures:
        1: keyword.other.change-list.git.commit
        2: punctuation.separator.mapping.pair.change-list.git.commit
        3: string.unquoted.git.commit
        4:  markup.changed.file.name.git.commit
        5: punctuation.separator.mapping.pair.path.git.commit
        6: markup.changed.file.name.git.commit
        7: markup.changed.file.name.git.commit
```

_Note: If we can ensure `typechange` to always contain a `->` the pattern could be simplified as follows:_

```YAML
    - match: \b({{copied}}|{{renamed}}|{{typechange}})\s*(:)\s*((.+)\s+(->)\s+(.+))
      scope: meta.change-list.git.commit
      captures:
        1: keyword.other.change-list.git.commit
        2: punctuation.separator.mapping.pair.change-list.git.commit
        3: string.unquoted.git.commit
        4:  markup.changed.file.name.git.commit
        5: punctuation.separator.mapping.pair.path.git.commit
        6: markup.changed.file.name.git.commit
```

Reply:
Unfortunately, `typechange` has no `->`.

![image](https://user-images.githubusercontent.com/6594915/65535106-5e4de480-df33-11e9-9db4-93a21d2e327f.png)

Reply:
Possible.

Reply:
I separate it into another rule. :+1: